decidim-admin/app/views/layouts/decidim/admin/_header.html.erb

```diff
@@ -3,6 +3,7 @@
 <%= legacy_favicon %>
 <%= favicon %>
 <%= apple_favicon %>
-<%= stylesheet_pack_tag "decidim_core", "decidim_admin", media: "all" %>
+<%= append_stylesheet_pack_tag "decidim_admin_overrides", media: "all" %>
+<%= stylesheet_pack_tag "decidim_core","decidim_admin", media: "all" %>
```

Comment:
Appends should be after the appended file if possible. Also there's a missing whitespace after the comma 
```suggestion
<%= stylesheet_pack_tag "decidim_core", "decidim_admin", media: "all" %>
<%= append_stylesheet_pack_tag "decidim_admin_overrides", media: "all" %>
```

Reply:
Actually, the "append_stylesheet_pack_tag" needs to be called *before* the `stylesheet_pack_tag` helper.
https://github.com/shakacode/shakapacker/blob/main/README.md#view-helper-append_javascript_pack_tag-prepend_javascript_pack_tag-and-append_stylesheet_pack_tag